packages/ngx-forms/src/lib/datepicker/components/datepicker/datepicker.component.ts

```diff
@@ -109,6 +117,35 @@ export class DatepickerComponent implements OnInit, OnDestroy, ControlValueAcces
     this.componentDestroyed$.complete();
   }
 
+  private createInterval() {
+    if (!this.min && !this.max) {
+      return;
+    }
+    // Create an interval if min/max is filled in
+    let interval: Interval.IInterval<Moment>;
+    if (!!this.min && !!this.max) {
+      interval = IntervalBuilder.momentInterval(this.min ? new Moment(this.min) : null, this.max ? new Moment(this.max) : null)
+        .bounded()
+        .closedInterval()
+        .not()
+        .build();
+    } else if (!this.min) {
+      interval = IntervalBuilder.momentInterval(null, new Moment(this.max))
+        .unbounded()
+        .leftOpenInterval()
+        .not()
```

Comment:
waarom de not? doet dit dan niet het omgekeerde van wat het zou moeten?

Btw, ik vind nog steeds dat deze if/else logica thuis hoort in ofwel IntervalBuilder.momentInterval ofwel in een MomentInterval constructor (waardoor de IntervalBuilder zelfs niet meer nodig is). Dat na het oproepen van IntervalBuilder.momentInterval het soms in een invalid state belandt tot een aantal extra methods opgeroepen worden vooraleer build() mag gedaan worden is bedenkelijk.

Reply:
De not() is gewoon de interpretatie van het interval zelf. Je geeft data in waarop je wél wilt selecteren en moet het interval omgedraait worden. Dit staat in contrast met de range, waarop je data wilt uischakelen. De component zelf ondersteunt eigenlijk alleen maar data die je wilt uitschakelen, dus als je min/max doorgeeft (de data die je net wél actief moet hebben), moet dat gereversed worden.